pkg/plugins/manager/signature/manifest.go

```diff
@@ -17,12 +17,9 @@ import (
 
 	"github.com/gobwas/glob"
 
-	// TODO: replace deprecated `golang.org/x/crypto` package https://github.com/grafana/grafana/issues/46050
-	// nolint:staticcheck
-	"golang.org/x/crypto/openpgp"
-	// nolint:staticcheck
-	"golang.org/x/crypto/openpgp/clearsign"
-
+	"github.com/ProtonMail/go-crypto/openpgp"
+	"github.com/ProtonMail/go-crypto/openpgp/clearsign"
+	"github.com/ProtonMail/go-crypto/openpgp/packet"
```

Comment:
Nit: can you move those in the same block as `gobaws/glob` as that's an external package?

```go
// ...
	"runtime"
	"strings"

	"github.com/gobwas/glob"
	"github.com/ProtonMail/go-crypto/openpgp"
	"github.com/ProtonMail/go-crypto/openpgp/clearsign"
	"github.com/ProtonMail/go-crypto/openpgp/packet"

	"github.com/grafana/grafana/pkg/plugins"

// etc etc
```